include/mrdox/Metadata/Access.hpp

```diff
@@ -44,6 +44,16 @@ struct MemberRef
 {
     SymbolID id;
     Access access;
+
+    constexpr MemberRef() = default;
+
+    constexpr MemberRef(
+        const SymbolID& id,
```

Comment:
please don't use the same formal parameter names as data members, change this to `id_`